ios/FluentUI/DrawerVnext/SlideOverPanel.swift

```diff
@@ -0,0 +1,174 @@
+/// `View` consist of content view placed on top on panel and transparent background layer
+struct SlideOverPanel<Content: View>: View {
+
+    // Width occupied by content and spacer combined
+    internal var slideOutPanelWidth: CGFloat = UIScreen.main.bounds.width
+
+    // action executed with background transperent view is tapped
+    internal var actionOnBackgroundTap: (() -> Void)?
+
+    // content view is visible when slide over panel is expanded
+    internal var content: Content
+
+    // opacity used to dim the transparent layer
+    internal var backgroundLayerOpacity: Double = 0.0
+
+    // slide out direction
+    internal var direction: SlideOverDirection = .left
+
+    // content is visible when set to `true`
+    @Binding internal var isOpen: Bool
+
+    // width content view needs to be occupied
+    @Binding internal var preferredContentOffset: CGFloat?
+
+    // configure the apperance of drawer
+    @ObservedObject public var tokens = DrawerTokens()
+
+    private let backgroundLayerColor: Color = .black
+
+    private let contentWidthSizeRatio: CGFloat = 0.9
+
+    public var body: some View {
+        HStack {
+            if direction == .right {
+                InteractiveSpacer().onTapGesture (count: 1, perform: actionOnBackgroundTap ?? {})
+            }
+
+            content
+                .frame(width: contentWidth())
+                .shadow(color: tokens.shadowColor.opacity(isOpen ? tokens.shadowOpacity : 0),
+                        radius: tokens.shadowBlur,
+                        x: tokens.shadowDepthX,
+                        y: tokens.shadowDepthY)
+                .offset(x: resolvedContentOffset())
+
+            if direction == .left {
+                InteractiveSpacer().onTapGesture (count: 1, perform: actionOnBackgroundTap ?? {})
+            }
+        }
+        .background(isOpen ? backgroundLayerColor.opacity(backgroundLayerOpacity) : Color.clear)
+    }
+
+    private func resolvedContentOffset() -> CGFloat {
+        // override offset if required
+        if let preferredContentWidth = preferredContentOffset {  // parent view wants to take over primarily to conform to user drag gesture
+            return preferredContentWidth
+        }
+
+        let offset = isOpen ? expandedConentOffset() : collapsedContentOffset()
+        if direction == .left {
+            return -offset
+        } else {
+            return offset
+        }
+    }
+
+    private func expandedConentOffset() -> CGFloat {
+        return CGSize.zero.width
+    }
+
+    private func collapsedContentOffset() -> CGFloat {
+        return slideOutPanelWidth * contentWidthSizeRatio
+    }
+
+    private func contentWidth() -> CGFloat {
+        return slideOutPanelWidth * contentWidthSizeRatio
+    }
+}
+
+// MARK: - Composite View
+
+public struct InteractiveSpacer: View {
+    public var body: some View {
+        ZStack {
+            Color.black.opacity(0.001)
```

Comment:
I think the same applies here for a tokenized opacity that can be tracked in the same issue I mentioned before.
Could you talk to Victor about it?

Reply:
Victor is OOF in the meanwhile I have added a transparent color to stylesheet. I will discuss this with him as a part of #358